### PR TITLE
"OpenVSM Virtual USB HID" PID request

### PR DESCRIPTION
**Pull request by Pugnator**
This PID is need for Virtual USB device. 
It will be used during simulation under Proteus for interracting with user's real hardware.

### CODE REVIEW

File: 1209/C0DE/index.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: OpenVSM Virtual USB
+owner: NonameGarage
```
Comment (Arachnid) [inline 1209/C0DE/index.md]: This needs to have the same capitalisation as the directory you put your org file in.

**Comment by Arachnid** [comment]
Thanks for your pull request! Can you clarify how this will be used? Will the user's operating system recognise this as a USB device? Will it have to load drivers? Would the test PID, http://pid.codes/1209/0001/ be suitable for this application?

**Comment by Pugnator** [comment]
Thanks a lot for your reply!
The main idea of the project is that you can simulate IC by means of Lua script. Current default realization of USB stack available in Proteus is closed source one. Users will be able to see theirs virtual devices in devmgr and use it as it is a real one. And in this case yes, 0001 may be suitable but also I wanted to use it in several other projects our group is released, for example this openhardware/software GSM/GPS tracker for autosport I'm working on
https://github.com/Pugnator/ATRS
It will be nice to have the same PID everywhere. Sorry for my english, I hope I was able to clarify my intentions
I fixed capitalization issue by the way.

**Comment by Arachnid** [comment]
Thanks for the details. It's generally a good idea to use a unique PID for each unique device that will be distributed to multiple users. As such, I'd recommend using our test PID 0x0001 for your virtual devices, and applying for a PID from us for anything that you expect to be distributed widely and require driver support.

**Pull request closed without merging**